tests/testthat/test-all.R

```diff
@@ -3,18 +3,29 @@ library(RSocrata)
 library(httr)
 library(jsonlite)
 library(mime)
+library(plyr)
```

Comment:
`plyr` is not needed for the tests.  If it's a package dependency it belongs in the appropriate places in the NAMESPACE and DESCRIPTION files.